23homework.sql

```diff
@@ -0,0 +1,108 @@
+-- Tworzy tabelę pracownik(imie, nazwisko, wyplata, data urodzenia, stanowisko). W tabeli mogą być dodatkowe kolumny, które uznasz za niezbędne.
+CREATE TABLE pracownicy (
+	id_pracownika int primary key auto_increment not null,
+    imie varchar(20) NOT NULL,
+    nazwisko varchar(50) NOT NULL,
```

Comment:
Nie ma co sie czaic z dlugoscia znakow, jesli nie masz super konkretnych potrzeb (np. wiesz ze PESEL ma zawsze taka sama liczbe znakow) to dawaj 255 :D